translations/introduction.tex/pl.po

```diff
@@ -39,6 +39,12 @@ msgid ""
 "several different Factions, led by their Heroes, battle for supremacy. "
 "Choose your Faction and Hero and banish your unruly enemies from these lands!"
 msgstr ""
+"\\hypertarget{Heroes of Might and Magic III}{\\textbf{Heroes of Might and "
+"Magic III: The Board Game}} jest taktyczno-strategiczną grą planszową, która "
+"w wersji core pozwala na rozgrywkę w gronie 1-3 osób. "
```

Comment:
```suggestion
"w wersji podstawowej pozwala na rozgrywkę w gronie 1-3 osób. "
```
A to nie tak?

Reply:
fakycznie - księga zasad też ma wzmiankę o zestaiwe podstawowym. zmienię to trochę jeszcze

Reply:
ok - dodałem zestaw